src/utils.ts

```diff
@@ -216,21 +216,14 @@ export const fetch = ({
     headers: {[key: string]: string};
     body: string;
 }): Promise<NativeAppResponsePayload<'FETCH'>> => {
-    if (isWebViewBridgeAvailable()) {
```

Comment:
this wrapping is not needed. The postMessage call fails with the same status if the bridge is not available